src/plots/FilledBoundary/FilledBoundaryPluginInfo.h

```diff
@@ -3,7 +3,7 @@
 // details.  No copyright assignment is required to contribute to VisIt.
 
 // ************************************************************************* //
-//                     FilledBoundaryPluginInfo.h                            //
+//                               FilledBoundaryPluginInfo.h                            //
```

Comment:
I wonder if it would be better if our code gen for this would output:
<pre>
// **********************************************************
//  FilledBoundaryPluginInfo.h
// **********************************************************
</pre>

instead of trying to 'center' the plugin name and having our end // not align ?
Since you are regenerating all the info objects, it would be a good time to make the change.

Reply:
I agree, can we add a new ticket for this change?

Reply:
Done #4177

Reply:
Ok, forget my previous comment on this issue in another file. I like the new idea of NOT centering these.

Reply:
thanks for adding #4177